### ISSUE TITLE
Error in GANInference#initilize() with logdir

### ISSUE THREAD

**Issue by tatsuhiko-inoue** [blei-lab/edward#716]
Hi.

When using GANInference and setting a value to "logdir" argument of "initilize" method, I got the following error:

```
Traceback (most recent ca｀ll last):
  File "gan_logdir.py", line 32, in <module>
    inference.initialize(logdir='log')
  File "/home/tatuhiko/local/virtualenv/tensorflow/local/lib/python2.7/site-packages/edward/inferences/gan_inference.py", line 94, in initialize
    self.build_loss_and_gradients(var_list)
  File "/home/tatuhiko/local/virtualenv/tensorflow/local/lib/python2.7/site-packages/edward/inferences/gan_inference.py", line 124, in build_loss_and_gradients
    tf.concat(d_true, d_fake, axis=0),
TypeError: concat() got multiple values for keyword argument 'axis'
```

The source code to occur this error is shown below:
```python
#!/usr/bin/env python

import numpy as np
import edward as ed
import tensorflow as tf
from tensorflow.contrib import slim
from edward.models import Normal

def generative_network():
  z = Normal(loc=tf.zeros([M, N]), scale=tf.ones([M, N]))
  h0 = slim.fully_connected(z, H, activation_fn=tf.nn.relu)
  h1 = slim.fully_connected(h0, 1, activation_fn=None)
  return h1

def discriminative_network(x):
  h0 = slim.fully_connected(x, H * 2, activation_fn=tf.tanh)
  h1 = slim.fully_connected(h0, H * 2, activation_fn=tf.tanh)
  h2 = slim.fully_connected(h1, H * 2, activation_fn=tf.tanh)
  h3 = slim.fully_connected(h2, 1, activation_fn=None)
  return h3

ed.set_seed(42)

M = 12  # batch size during training
H = 4  # number of hidden units
N = 8

x_ph = tf.placeholder(tf.float32, [M, 1])
x = generative_network()

inference = ed.GANInference(data={x: x_ph}, discriminator=discriminative_network)
inference.initialize(logdir='log')

tf.global_variables_initializer().run()

for _ in range(inference.n_iter):
    x_data = np.zeros([M, 1])
    inference.update(feed_dict={x_ph: x_data})
```

**Comment by dustinvtran** [comment]
Good catch! This is a bug: the line should be `tf.concat([d_true, d_fake], axis=0)`. Contribution welcome.

**Comment by dustinvtran** [comment]
fixed in #718
